comit/src/expiries.rs

```diff
@@ -35,104 +40,140 @@ pub struct Expiries<A, B> {
     alpha_connector: A,
     /// Beta blockchain connector for getting the current time.
     beta_connector: B,
-    /// The alpha ledger relative expiry time.
-    alpha_expiry: Duration,
-    /// The beta ledger relative expiry time.
-    beta_expiry: Duration,
+    /// The alpha ledger expiry offset.
+    alpha_offset: AlphaOffset,
+    /// The beta ledger expiry offset.
+    beta_offset: BetaOffset,
+}
+
+/// Calculate a pair of useful expiries, 'useful' means a swap can be
+/// successfully completed with these expiries.
+/// Multiply the resulting offsets by `scale_factor` percentage to give the
+/// counterparty more time to act ( <100 shrinks offsets, >100 expands offsets).
+pub fn calculate_expiry_offsets(
```

Comment:
I am unable to come up with any tests that test this since 'as expected' is hard to define. We have a function that returns a 'report' string of the expiries created and when the unit tests are run with `--nocapture` this report can be inspected. I'm open to suggestions but thus far I have not been able to come up with anything better.